### PR TITLE
Added Layout Keymaps for BriianPowell

### PR DESCRIPTION
**Pull request by BriianPowell**
## Description

Added Layout Keymaps for BriianPowell:
- Discipline
- Gentleman65

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* For those wondering if the Gentleman works well with QMK, there is full support I have not had any issues

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
It looks like you have changes to the submodules here.  These need to be reverted before the PR can be approved. 

Rebasing sometimes will fix this. Otherwise, you'd want to do something like this to fix them:
```sh
git checkout master
git submodule foreach 'git log -n1 --oneline'
# remember the first abcdef1234 on the submodules that have an issue
git checkout @@your-branch@@
# for each of the problematic repo's:
cd lib/@@name@@
git checkout abcdef1234
```

**Comment by BriianPowell** [comment]
@drashna I updated the PR to remove the changes to the submodules can you please review again :)

**Review by drashna — APPROVED** [review]

File: keyboards/coseyfannitutti/discipline/keymaps/briianpowell/keymap.c [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+     * |---------------------------------------------------------------|
+     * |    |    |    |      PLY/PS      |    |    |    |  |PRV|VL-|NXT|
+     * `---------------------------------------------------------------'
+     */
+    [_FUNC] = LAYOUT_65_ansi( \
+      _______,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,_______,   RESET,\
+      _______,KC_WH_U,KC_BTN1,KC_MS_U,KC_BTN2,_______,_______,AG_NORM,AG_SWAP,QWERTY,WORKMAN,_______,_______,_______,   KC_INS,\
+      _______,KC_WH_D,KC_MS_L,KC_MS_D,KC_MS_R,_______,_______,_______,_______,_______,_______,_______,       _______,  KC_PGUP,\
+      _______,KC_WH_L,KC_BTN3,KC_WH_R,_______,_______,_______,_______,_______,_______,_______,        _______,KC_VOLU, KC_PGDN,\
+      _______,_______,_______,                        KC_MPLY,                _______,_______,_______,KC_MPRV,KC_VOLD, KC_MNXT \
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/discipline/keymaps/briianpowell/keymap.c:94]:
```suggestion
    [_FUNC] = LAYOUT_65_ansi(
      _______,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,_______,   RESET,\
      _______,KC_WH_U,KC_BTN1,KC_MS_U,KC_BTN2,_______,_______,AG_NORM,AG_SWAP,QWERTY,WORKMAN,_______,_______,_______,   KC_INS,
      _______,KC_WH_D,KC_MS_L,KC_MS_D,KC_MS_R,_______,_______,_______,_______,_______,_______,_______,       _______,  KC_PGUP,
      _______,KC_WH_L,KC_BTN3,KC_WH_R,_______,_______,_______,_______,_______,_______,_______,        _______,KC_VOLU, KC_PGDN,
      _______,_______,_______,                        KC_MPLY,                _______,_______,_______,KC_MPRV,KC_VOLD, KC_MNXT
```

File: keyboards/coseyfannitutti/discipline/keymaps/briianpowell/keymap.c [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+   * |---------------------------------------------------------------|
+   * |Ctrl|Alt |Gui |         Space    |Gui |Alt |Ctrl|  |Lef|Dow|Rig|
+   * `---------------------------------------------------------------'
+   */
+  [_WORKMAN] = LAYOUT_65_ansi( \
+      KC_ESC,  KC_1,   KC_2,   KC_3, KC_4, KC_5, KC_6, KC_7, KC_8,    KC_9,   KC_0,    KC_MINS,  KC_EQL, KC_BSPC, KC_GRV,\
+      KC_TAB,  KC_Q,   KC_D,   KC_R, KC_W, KC_B, KC_J, KC_F, KC_U,    KC_P,   KC_SCLN, KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,\
+      FN_ESC,  KC_A,   KC_S,   KC_H, KC_T, KC_G, KC_Y, KC_N, KC_E,    KC_O,   KC_I,    KC_QUOT,          KC_ENT, KC_HOME,\
+      KC_LSFT,  KC_Z,   KC_X,   KC_M, KC_C, KC_V, KC_K, KC_L, KC_COMM, KC_DOT, KC_SLSH,         KC_RSFT,  KC_UP,   KC_END,\
+      KC_LCTL,  KC_LGUI,KC_LALT,            KC_SPC,                  KC_RALT, KC_RGUI, KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT \
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/discipline/keymaps/briianpowell/keymap.c:73]:
```suggestion
  [_WORKMAN] = LAYOUT_65_ansi(
      KC_ESC,  KC_1,   KC_2,   KC_3, KC_4, KC_5, KC_6, KC_7, KC_8,    KC_9,   KC_0,    KC_MINS,  KC_EQL, KC_BSPC, KC_GRV,
      KC_TAB,  KC_Q,   KC_D,   KC_R, KC_W, KC_B, KC_J, KC_F, KC_U,    KC_P,   KC_SCLN, KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL,
      FN_ESC,  KC_A,   KC_S,   KC_H, KC_T, KC_G, KC_Y, KC_N, KC_E,    KC_O,   KC_I,    KC_QUOT,          KC_ENT, KC_HOME,
      KC_LSFT,  KC_Z,   KC_X,   KC_M, KC_C, KC_V, KC_K, KC_L, KC_COMM, KC_DOT, KC_SLSH,         KC_RSFT,  KC_UP,   KC_END,
      KC_LCTL,  KC_LGUI,KC_LALT,            KC_SPC,                  KC_RALT, KC_RGUI, KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```

File: keyboards/coseyfannitutti/discipline/keymaps/briianpowell/keymap.c [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  * |---------------------------------------------------------------|
+  * |Ctrl|Gui |Alt |         Space    |Gui |Alt |Ctrl|  |Lef|Dow|Rig|
+  * `---------------------------------------------------------------'
+  */
+  [_QWERTY] = LAYOUT_65_ansi( \
+    KC_ESC,  KC_1,    KC_2,    KC_3,  KC_4, KC_5, KC_6, KC_7, KC_8,   KC_9,  KC_0,   KC_MINS, KC_EQL, KC_BSPC,  KC_GRV,\
+    KC_TAB,  KC_Q,    KC_W,    KC_E,  KC_R, KC_T, KC_Y, KC_U, KC_I,   KC_O,  KC_P,   KC_LBRC,KC_RBRC, KC_BSLS,  KC_DEL,\
+    FN_ESC,  KC_A,    KC_S,    KC_D,  KC_F, KC_G, KC_H, KC_J, KC_K,   KC_L,  KC_SCLN,KC_QUOT,         KC_ENT,  KC_HOME,\
+    KC_LSFT,  KC_Z,    KC_X,    KC_C,  KC_V, KC_B, KC_N, KC_M, KC_COMM,KC_DOT,KC_SLSH,        KC_RSFT, KC_UP,    KC_END,\
+    KC_LCTL,  KC_LGUI, KC_LALT,            KC_SPC,                  KC_RALT, KC_RGUI, KC_RCTL,KC_LEFT, KC_DOWN, KC_RGHT \
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/discipline/keymaps/briianpowell/keymap.c:52]:
```suggestion
  [_QWERTY] = LAYOUT_65_ansi(
    KC_ESC,  KC_1,    KC_2,    KC_3,  KC_4, KC_5, KC_6, KC_7, KC_8,   KC_9,  KC_0,   KC_MINS, KC_EQL, KC_BSPC,  KC_GRV,
    KC_TAB,  KC_Q,    KC_W,    KC_E,  KC_R, KC_T, KC_Y, KC_U, KC_I,   KC_O,  KC_P,   KC_LBRC,KC_RBRC, KC_BSLS,  KC_DEL,
    FN_ESC,  KC_A,    KC_S,    KC_D,  KC_F, KC_G, KC_H, KC_J, KC_K,   KC_L,  KC_SCLN,KC_QUOT,         KC_ENT,  KC_HOME,
    KC_LSFT,  KC_Z,    KC_X,    KC_C,  KC_V, KC_B, KC_N, KC_M, KC_COMM,KC_DOT,KC_SLSH,        KC_RSFT, KC_UP,    KC_END,
    KC_LCTL,  KC_LGUI, KC_LALT,            KC_SPC,                  KC_RALT, KC_RGUI, KC_RCTL,KC_LEFT, KC_DOWN, KC_RGHT
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/coseyfannitutti/discipline/keymaps/briianpowell/keymap.c [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+     * |---------------------------------------------------------------|
+     * |      |   |   |   |   |   |   |   |   |   |   |   |        |HME|
+     * |---------------------------------------------------------------|
+     * |    |   |   |   |   |   |   |   |   |   |   |   |      |VL+|END|
+     * |---------------------------------------------------------------|
+     * |    |    |    |      PLY/PS      |    |    |    |  |PRV|VL-|NXT|
+     * `---------------------------------------------------------------'
+     */
+    [_FUNC] = LAYOUT_65_ansi(
+      _______,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,_______,   RESET,\
```
Comment (fauxpark) [inline keyboards/coseyfannitutti/discipline/keymaps/briianpowell/keymap.c:90]:
```suggestion
      _______,  KC_F1,  KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9, KC_F10, KC_F11, KC_F12,_______,   RESET,
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by BriianPowell** [comment]
@fauxpark I have commited the requested changes can you plaese review again? :)

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**